CorsixTH/Lua/dialogs/error_confirm.lua

```diff
@@ -0,0 +1,123 @@
+--[[ Copyright (c) 2009 Peter "Corsix" Cawley
```

Comment:
This line looks wrong unless you live in the past and impersonate someone else :)